docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -0,0 +1,561 @@
+  from one of its family targets.
+* `tfm_target_name`: Target name in TF-M.
+* `tfm_bootloader_supported`: If TF-M bootloader is supported by the target.
+
+    The supported values are `true` and `false`.
+* `tfm_supported_toolchains`: Supported TF-M toolchains.
+
+    The supported values are `ARMCLANG` and `GNUARM`.
+* `tfm_default_toolchain`: Default TF-M toolchain.
+
+    The supported values are `ARMCLANG` and `GNUARM`.
+* `tfm_delivery_dir`: The directory to which TF-M binaries will be copied.
+* `TFM_OUTPUT_EXT`: Optional attribute that indicates the output extension of the TF-M secure binary.<!--how come only this one is all caps?-->
+
+The following example shows a PSA enabled dual-core target, `PSoC64`:
+
+```json
+    "CY8CKIT_064S2_4343W": {
+        "inherits": [
+            "MCU_PSOC6_M4"
+        ],
+        "features_add": [
+            "BLE",
+            "PSA"
+        ],
+        "components_add": [
+            "WHD",
+            "4343W",
+            "CYW43XXX"
+        ],
+        "components_remove": [
+            "QSPIF"
+        ],
+        "supported_form_factors": [
+            "ARDUINO"
+        ],
+        "device_has_remove": [
+            "ANALOGOUT",
+            "QSPI"
+        ],
+        "extra_labels_add": [
+            "PSOC6_02",
+            "MXCRYPTO_02",
+            "CORDIO",
+            "TFM",
+            "TFM_DUALCPU"
+        ],
+        "macros_add": [
+            "CYB0644ABZI_S2D44",
+            "CYBSP_WIFI_CAPABLE",
+            "TFM_MULTI_CORE_MULTI_CLIENT_CALL=1",
+            "MBEDTLS_PSA_HAS_ITS_IO",
+            "MBEDTLS_USE_PSA_CRYPTO"
+        ],
+        "detect_code": [
+            "190A"
+        ],
+        "post_binary_hook": {
+            "function": "PSOC6Code.sign_image"
+        },
+        "forced_reset_timeout": 5,
+        "overrides": {
+            "network-default-interface-type": "WIFI"
+        },
+        "program_cycle_s": 10,
+        "tfm_target_name": "psoc64",
+        "tfm_bootloader_supported": false,
+        "tfm_default_toolchain": "GNUARM",
+        "tfm_supported_toolchains": [
+            "GNUARM"
+        ],
+        "tfm_delivery_dir": "TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W",
+        "TFM_OUTPUT_EXT": "hex"
+    }
+```
+
+Please note the config options `macros_add`, `extra_labels_add` and `device_has_remove`. To add or remove `macros`, `extra_labels` or target capabilities, a PSA target definition must use
+[macros/extra_labels/device_has]`_add` or [macros/extra_labels/device_has]`_remove` (not `macros`, `extra_labels` or `device_has`).
+
+To add or remove a feature, use `feature_`[add/remove].
+
+By default, a TF-M build generates a `bin` file. If the target requires a `hex` file, you need to add the attribute `"TFM_OUTPUT_EXT": "hex"` to the target definition. The build script will convert `bin` to `hex`, then copy the `hex` to `tfm_delivery_dir`.
```

Comment:
```suggestion
By default, a TF-M build generates a `bin` file. If the target requires a `hex` file, you need to add the attribute `"tfm_output_ext": "hex"` to the target definition. The build script will convert `bin` to `hex`, then copy the `hex` to `tfm_delivery_dir`.
```

Reply:
Ah, I misunderstood you - I thought you meant it was an error in the docs, not the code ;)

Reply:
Unfortunately, we can't change it here. The code contains capitalised text.